src/renderer/scss/component/_tabs.scss

```diff
@@ -50,8 +50,13 @@ nav.sub-header
       color: var(--tab-active-color);
     }
   }
+
+  &.sub-header--full-width {
```

Comment:
I have some non-blocking concerns about this, but will wait to discuss them on a meeting.